src/SFA.DAS.CaptureTheFlag.Web/Startup.cs

```diff
@@ -36,7 +40,15 @@ public void ConfigureServices(IServiceCollection services)
             services.AddSingleton<IGameRepository>(new GameRepository());
             services.AddSignalR();
             services.AddRazorPages();
+            services.AddServerSideBlazor();
```

Comment:
We're using Blazor?

Reply:
I've dropped a message in the slack channel